src/formatMessagesAsString/index.js

```diff
@@ -0,0 +1,32 @@
+import chalk from 'chalk'
+
+/**
+ * @param {Array<Object>} messages: An array of message objects.
+ * @returns {String} A colored string representation of the given message objects.
+ */
+export default function formatMessagesAsString(messages) {
+  // Return undefined when there are no messages to format
+  if (!messages.length) {
+    return undefined
+  }
+
+  return messages.reduce(messageReducer, '')
+
+  function messageReducer(accumulatorString, messageObject, index, array) {
+    // Don't append newline to the last entry
+    const newline = index === array.length - 1 ? '\n' : ''
+
+    const formattedMessage = formatMessage(messageObject)
+    return `${accumulatorString}${newline}${formattedMessage}\n`
+  }
+}
```

Comment:
Cool, I was wondering, to get away from maintaining `\n`logic.

Thanks for sharing your views.